moby/moby
# Add monitored resource type metadata for GCP logging driver

**- What I did**

Added metadata about monitored resource type for GCP logging driver.

**- How I did it**

I used GCP logging SDK.

**- How to verify it**

Run a docker container using GCP logging driver on GCE VM and go to GCE VM logs, observe that logs are attributed to VM rather than being logged to global sink. Example to run a command could be
` docker run --log-driver=gcplogs docker.io/tomcat `

Before this change:
![gcplogging-before](https://cloud.githubusercontent.com/assets/2766140/25551470/a9228d0c-2c85-11e7-9934-f1d737bd8ac6.png)

After this change:
![gcplogging-after](https://cloud.githubusercontent.com/assets/2766140/25551471/b12e3d0c-2c85-11e7-8da7-66b07c57d1db.png)

**- Description for the changelog**
Add monitored resource type metadata for GCP logging driver

**- A picture of a cute animal (not mandatory but encouraged)**
![hamster](https://cloud.githubusercontent.com/assets/2766140/25551534/7d230c8a-2c86-11e7-98ba-b00b63f94ea7.jpg)
